RiotSwiftUI/Modules/Spaces/SpaceCreation/SpaceCreationEmailInvites/Service/MatrixSDK/SpaceCreationEmailInvitesService.swift

```diff
@@ -42,13 +42,14 @@ class SpaceCreationEmailInvitesService: SpaceCreationEmailInvitesServiceProtocol
     }
 
     func prepareIdentityService(prepared: ((String?, String?) -> Void)?, failure: ((Error?) -> Void)?) {
-        isLoadingSubject.send(true)
-        session.prepareIdentityServiceForTerms(withDefault: RiotSettings.shared.identityServerUrlString) { [weak self] session, baseURL, accessToken in
-            self?.isLoadingSubject.send(false)
-            prepared?(baseURL, accessToken)
-        } failure: { [weak self] error in
-            self?.isLoadingSubject.send(false)
-            failure?(error)
-        }
+        // Tchap: Disable Spaces
+//        isLoadingSubject.send(true)
```

Comment:
Could we ignore this file in project ?

Reply:
I cannot change this, because this change is mandatory to build the project, and this file cannot be removed from the target. There are a lots of dependencies with the Spaces feature everywhere inside the code.